#List of port IDs
# port_id_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 port_Current  = 0 #  Configure the port which receives the 0x1005 message
	uint8 port_COM1     = 1
	uint8 port_COM2     = 2
	uint8 port_COM3     = 3
	uint8 port_COM4     = 4
	uint8 port_SPI      = 5
	uint8 port_USB      = 7
	uint8 port_ETHERNET = 9
	uint8 port_CAN      = 12
